README.md

```diff
@@ -2,4 +2,135 @@
 
 [![Build Status](https://travis-ci.org/AmazeeLabs/delivery.svg?branch=8.x-1.x)](https://travis-ci.org/AmazeeLabs/delivery) [![codecov](https://codecov.io/gh/AmazeeLabs/delivery/branch/8.x-1.x/graph/badge.svg)](https://codecov.io/gh/AmazeeLabs/delivery) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/AmazeeLabs/delivery/badges/quality-score.png?b=8.x-1.x)](https://scrutinizer-ci.com/g/AmazeeLabs/delivery/?branch=8.x-1.x)
 
-Readme coming soon.
+# Module dependencies:
```

Comment:
This can sometimes get outdated, we could point to the module yaml file instead, or as a source of truth checking but leave current list as easy to glance over for people interested in the module.

Reply:
I've updated the list for now.